react-graphql-apollo-tutorial.md

```diff
@@ -456,22 +456,22 @@ const client = new ApolloClient({
 
 ### Exercises:
 
-* read more about {{% a_blank "the network layer configuration in Apollo Client" "https://www.apollographql.com/docs/react/advanced/network-layer.html" %}}
-* invest 3 minutes of your time and take the {{% a_blank "quiz" "https://www.surveymonkey.com/r/5FYZT8T" %}}
+* Read more about {{% a_blank "the network layer configuration in Apollo Client" "https://www.apollographql.com/docs/react/advanced/network-layer.html" %}}
+* Invest 3 minutes of your time and take the {{% a_blank "quiz" "https://www.surveymonkey.com/r/5FYZT8T" %}}
 
 {{% chapter_header "Connect Data-Layer to View-Layer: Introducing React Apollo" "react-apollo-connecting-layers" %}}
 
-Everything you have done before was the framework agnostic part of Apollo Client. However, without connecting React to the Apollo Client, you would have a hard time making use of GraphQL in React in a convenient way. That's why there exists an official library to connect both worlds: {{% a_blank "react-apollo" "https://github.com/apollographql/react-apollo" %}}. The great thing about those connecting libraries: there exist solutions for other view-layer solutions such as Angular and Vue too. Hence you can make use of the Apollo Client in a framework agnostic way. In the following, it needs two steps to connect the Apollo Client with React. First, install the mentioned library on the command line in your project folder:
+All we've donen thus far has been the framework agnostic part of Apollo Client. However, without connecting React to it, you'd have a hard time making effective use of GraphQL. That's why there is an official library to connect both worlds: {{% a_blank "react-apollo" "https://github.com/apollographql/react-apollo" %}}. The great thing about those connecting libraries is that there are solutions for other view-layer solutions like Angular and Vue, too, so you can use the Apollo Client in a framework agnostic way. In the following, it needs two steps to connect the Apollo Client with React. First, install the library in the command line in your project folder:
 
 {{< highlight javascript >}}
 npm install react-apollo --save
 {{< /highlight >}}
 
-Second, import its ApolloProvider component and use it as a composing component around your App component. Under the hood, it uses React's Context API to pass the Apollo Client through your application. You shouldn't forget to pass the client instance to it as prop:
+Second, import its ApolloProvider component, and use it as a composing component around your App component. Under the hood, it uses React's Context API to pass the Apollo Client through your application. Remember to pass the client instance to it as prop:
```

Comment:
"Under the hood, it uses React's Context API to pass the Apollo Client through your application." to "Under the hood, it uses [React's Context API](https://www.robinwieruch.de/react-context-api/) to pass the Apollo Client through your application."